client/trino-cli/src/test/java/io/trino/cli/TestAlignedTablePrinter.java

```diff
@@ -77,9 +77,7 @@ public void testAlignedPrinting()
     public void testHexPrintingInLists()
             throws Exception
     {
-        List<Column> columns = ImmutableList.<Column>builder()
-                .add(column("list", ARRAY))
```

Comment:
The good thing this approach is that when you add an entry you create new line so git blame is cleaner.

Reply:
Sure, I can revert some of them, but it's quite arbitrary where this pattern is appropriate

Reply:
I think we don't need to return any of these.